server/journeys/record-investigation/evidence-secured/schemas.ts

```diff
@@ -1,14 +1,14 @@
 import z from 'zod'
+import { createSchema } from '../../../middleware/validationMiddleware'
 
 const REASONS_MSG = 'Enter a description of the evidence secured'
 const TOO_LONG_ERROR_MSG = 'Description of the evidence secured must be 4,000 characters or less'
 
-export const schema = z.object({
+export const schema = createSchema({
   evidenceSecured: z
     .string({ message: REASONS_MSG })
     .max(4000, TOO_LONG_ERROR_MSG)
-    .trim()
-    .min(1, { message: REASONS_MSG }),
+    .refine(val => val?.trim().length > 0, { message: REASONS_MSG }),
```

Comment:
we're not transforming the user input.
if they want leading and following whitespaces in their input, we keep them.
trim() would transform the output value rather than just trimming it for validation.

Reply:
Oh, I thought we specifically didn't want those whitespaces.

Reply:
we asked https://mojdt.slack.com/archives/C06H6BHG2J3/p1717769273472779
and there wasn't a decision on trimming whitespaces

Reply:
got confirmation: https://mojdt.slack.com/archives/C06JRCZ5831/p1723032041968469 that we don't trim